packages/dd-trace/src/profiling/exporters/agent.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const axios = require('axios')
+const FormData = require('form-data')
+const { URL } = require('url')
+const { Encoder } = require('../encoders/pprof')
+const platform = require('../../platform')
+
+class AgentExporter {
+  constructor ({ url, hostname, port } = {}) {
+    url = new URL(url || `http://${hostname || 'localhost'}:${port || 8126}`)
+
+    this._client = axios.create({
+      baseURL: `${url}profiling/v1/`,
+      timeout: 10 * 1000,
+      validateStatus: code => code < 400
+    })
+
+    this._encoder = new Encoder()
+  }
+
+  async export ({ profiles, start, end, tags }) {
```

Comment:
Yes, although this was for the tracer and not the profiler, which used to be a separate library. I figured since it only exports every minute that it was fine to use promises, but I can change it if we prefer to be safe, especially now that both libraries are merging.